File: Swap.java
import java.util.Scanner;

public class Swap {
    public static int[] swap(int a, int b) {
        int temp; 
        temp = a;
        a = b;
        b = temp;
        return new int[]{a, b};
    }

    public static void main(String[] args) {
        Scanner inp = new Scanner(System.in);
        System.out.print("Enter the first value: ");
        int a = inp.nextInt();
        System.out.print("Enter the second value: ");
        int b = inp.nextInt();
        int[] swappedValues = swap(a, b);
        System.out.println("After swapping: a = " + swappedValues[0] + ", b = " + swappedValues[1]);
        inp.close();
    }
}

File: swap.py
def swap(a, b):
    swap_list = []
    temp, a, b = a, b, a # Swapping the values using variable unpacking
    swap_list.append(a)
    swap_list.append(b)
    return swap_list

def main():
    a = int(input("Enter first number: "))
    b = int(input("Enter second number: "))
    swap(a, b)
    for i in swap(a, b):
        print(i)

if __name__ == "__main__":
    main()